ang/test/civicase/Utils.spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env jasmine */
+(function ($) {
+  describe('Utils', function () {
+    var DateHelper, pastDate, futureDate;
+
+    beforeEach(module('civicase'));
+
+    describe('DateHelper', function () {
+      beforeEach(inject(function (_DateHelper_) {
+        DateHelper = _DateHelper_;
+      }));
+
+      describe('formatDate()', function () {
+        it('checks the format of the date', function () {
```

Comment:
Wrong description.
It does not check the format, it returns the date in the sent format.